src/lexedata/util.py

```diff
@@ -64,8 +64,12 @@ def normalize_string(text: str):
     return unicodedata.normalize("NFC", text.strip())
 
 
-def get_cell_comment(cell: op.cell.Cell) -> t.Optional[str]:
-    return cell.comment.text.strip() if cell.comment else ""
+def get_cell_comment(cell: op.cell.Cell) -> str:
+    raw_comment = cell.comment.text.strip() if cell.comment else ""
+    lines = [
+        line for line in raw_comment.split("\n") if line.strip() != "-lexedata.exporter"
```

Comment:
This line seems wrong to me. In the exporter/cognate we add the package name as the author of the comment.
Here we read the text of the comment, so the author, the package name, does not appear in comment.text.

Reply:
But somehow it does (probably due to some artefact of the loop), and then this is the right point to through it out, I hope?